src/fides/api/models/privacy_experience.py

```diff
@@ -432,10 +406,10 @@ def new_experience_needed(
         )
 
         privacy_center_notices: Query = get_privacy_notices_by_region_and_component(
-            db, region, ComponentType.privacy_center
+            db, [region.value], ComponentType.privacy_center
```

Comment:
I'm wondering if we need to pass in the region's country here too?  This function is making sure that a `PrivacyExperience` exists if a new region x component that didn't exist before was added to a Notice.  I just want to make sure we have "US" experiences too!

Reply:
yup i think you are right, i think this was a miss! not sure i gave it much though on the first pass through, i think i was trying to just touch as little as possible, but looking closer i don't see any reason why we _wouldn't_ also update this accordingly 👍